weblate/addons/forms.py

```diff
@@ -201,6 +201,20 @@ class JSONCustomizeForm(BaseAddonForm):
     )
 
 
+class XMLCustomizeForm(BaseAddonForm):
+    """Class defining user Form to configure XML Formatting AddOn"""
+
+    tags_format = forms.ChoiceField(
+        label=_("XML closing tags format"),
+        choices=[
+            ("self_closing_tags", _("Self-Closing Tags")),
+            ("closing_tags", _("Closing Tags")),
+        ],
+        required=True,
+        initial="self_closing_tags",
+    )
```

Comment:
This needs rephrasing so that it is easier to understand. Also, making it checkbox might be nicer.

How about this? (It would need changes in the add-on as well)

```suggestion
    closing_tags = forms.Boolean(
        label=_("Include closing tag for blank XML tags"),
        required=False,
        initial=True,
    )
```

Reply:
Definitely